### PR TITLE
cleanup(esm_runscripts/helpers.py): various cleanups

### PR DESCRIPTION
**Pull request by pgierz**
Most of this is just small formatting changes my editor kept annoying me
about. There are no behaviour changes.

### CODE REVIEW

File: src/esm_runscripts/helpers.py [hunk 1]
```diff
@@ -22,25 +22,24 @@ def evaluate(config, job_type, recipe_name):
...
         sys.exit(1)
 
     FUNCTION_PATH = esm_tools.get_config_filepath()
 
-    ###########################################################################################
+    ####################################################################################
     # LA: hotfix for non-matching python paths
     if (
-        "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"
+        "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"  # noqa: E501
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:34]: This just turns off the line length warning

File: src/esm_runscripts/helpers.py [hunk 2]
```diff
@@ -22,25 +22,24 @@ def evaluate(config, job_type, recipe_name):
...
         in sys.path
     ):
         sys.path.remove(
-            "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"
+            "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"  # noqa:501
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:38]: As above

File: src/esm_runscripts/helpers.py [hunk 3]
```diff
@@ -22,25 +22,24 @@ def evaluate(config, job_type, recipe_name):
             "general"
         ].get(recipe_name)
     except KeyError:
-        print(
-            f"Your configuration is incorrect, and should include headings for {setup_name} as well as general!"
-        )
+        print("Your configuration is incorrect.")
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:25]: Line length nonsense, in this case not important

File: src/esm_runscripts/helpers.py [hunk 4]
```diff
@@ -22,25 +22,24 @@ def evaluate(config, job_type, recipe_name):
...
+        print("It should include headings for the setup_name as well as general!")
         sys.exit(1)
 
     FUNCTION_PATH = esm_tools.get_config_filepath()
 
-    ###########################################################################################
+    ####################################################################################
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:31]: As above

File: src/esm_runscripts/helpers.py [hunk 5]
```diff
@@ -22,25 +22,24 @@ def evaluate(config, job_type, recipe_name):
...
         in sys.path
     ):
         sys.path.remove(
-            "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"
+            "/global/AWIsoft/tkleiner/lib/python2.7/site-packages/netCDF4-1.2.4-py2.7-linux-x86_64.egg"  # noqa:501
         )
     if "/global/AWIsoft/tkleiner/lib/python2.7/site-packages" in sys.path:
         sys.path.remove("/global/AWIsoft/tkleiner/lib/python2.7/site-packages")
-    ###########################################################################################
+    ####################################################################################
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:42]: As above

File: src/esm_runscripts/helpers.py [hunk 6]
```diff
@@ -63,9 +62,9 @@ def evaluate(config, job_type, recipe_name):
     return config
 
 
-#########################################################################################
-#                                   general stuff                                       #
-#########################################################################################
+########################################################################################
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:65]: As above

File: src/esm_runscripts/helpers.py [hunk 7]
```diff
@@ -84,19 +83,19 @@ def write_to_log(config, message, message_sep=None):
     message : list
         A list of the message elements; which is joined by either (highest
         to lowest): 1) the message_sep argument passed to the method, 2)
-        The user's chosen seperator, as written in
+        The user's chosen separator, as written in
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:86]: Small spelling mistakes

File: src/esm_runscripts/helpers.py [hunk 8]
```diff
@@ -84,19 +83,19 @@ def write_to_log(config, message, message_sep=None):
...
         ``config["general"]["experiment_log_file_message_sep"]``, 3)
         An empty space ``" "``.
     message_sep : None
-        The hard-coded message seperator to use; which ignores user choices.
+        The hard-coded message separator to use; which ignores user choices.
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:90]: As above

File: src/esm_runscripts/helpers.py [hunk 9]
```diff
@@ -84,19 +83,19 @@ def write_to_log(config, message, message_sep=None):
...
 
     Note
     ----
     The user can control two things regarding the logfile format:
 
     1) The datestamp formatting, whjich is taken from the config
        section ``general.experiment_log_file_dateformat``.
-    2) The message seperators; taken from
+    2) The message separators; taken from
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:98]: As above

File: src/esm_runscripts/helpers.py [hunk 10]
```diff
@@ -129,15 +128,15 @@ def assemble_log_message(
         timestampStr = "$(date +" + strftime_str + ")"
     else:
         timestampStr = dateTimeObj.strftime(strftime_str)
-    # TODO: Do we want to be able to specify a timestamp seperator as well?
+    # TODO: Do we want to be able to specify a timestamp separator as well?
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:131]: As above

File: src/esm_runscripts/helpers.py [hunk 11]
```diff
@@ -129,15 +128,15 @@ def assemble_log_message(
...
     line = timestampStr + " : " + message_sep.join(message)
     return line
 
 
 def update_reusable_filetypes(config, reusable_filetypes=None):
     """
     Removes some filetypes from the "resuable files" list. Filetypes to be
-    removed are either specifed in the config, or passed as an argument.
+    removed are either specified in the config, or passed as an argument.
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:139]: As above

File: src/esm_runscripts/helpers.py [hunk 12]
```diff
@@ -195,8 +194,8 @@ def update_reusable_filetypes(config, reusable_filetypes=None):
         if update_filetype not in potentially_reusable_filetypes:
             esm_parser.user_error(
                 "update-filetypes",
-                f"``{update_filetype}`` specified by you in ``--update-filetypes`` is not a "
-                + "ESM-Tools file type. Please, select one (or more) of the "
+                f"``{update_filetype}`` specified by you in ``--update-filetypes`` is "
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:197]: Line length

File: src/esm_runscripts/helpers.py [hunk 13]
```diff
@@ -227,7 +226,9 @@ def update_reusable_filetypes(config, reusable_filetypes=None):
     return reusable_filetypes
 
 
-############################## SINK CLASS FOR LOGURU.LOGGER ###########################
+##############################
+# SINK CLASS FOR LOGURU.LOGGER
+##############################
```
Comment (pgierz) [inline src/esm_runscripts/helpers.py:231]: Line length

**Review by pgierz — COMMENTED** [review]
Here are some small comments about my changes. This is just to get my editor to stop bothering me.

**Review by mandresm — APPROVED** [review]

**Comment by pgierz** [comment]
#bump

**Pull request merged**